Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 as builder
+
+COPY . /ModShark
+WORKDIR /ModShark
+
+RUN dotnet publish ModShark.sln
+
+
+FROM mcr.microsoft.com/dotnet/runtime:8.0
+
+# Install jq and use Docker caching for package downloads
+ENV DEBIAN_FRONTEND=noninteractive
+RUN --mount=target=/var/lib/apt/lists,type=cache,sharing=locked \
+    --mount=target=/var/cache/apt,type=cache,sharing=locked \
+    rm -rf /etc/apt/apt.conf.d/docker-clean && \
+    apt-get update && apt-get install -y jq
+
+# Add modshark user and copy built modshark
+RUN useradd -m -s /bin/bash modshark
+COPY --from=builder --chown=modshark:modshark /ModShark/ModShark/bin/Release/net8.0/publish/ /ModShark
```

Comment:
If it helps, there's now a [`/Resources/build-prod.ps1` script](https://github.com/warriordog/ModShark/blob/main/Resources/build-prod.ps1) that runs a build and stages all the compiled output in a predictable `/Publish` directory. You can copy the whole thing as-is or just take parts.

Reply:
Awesome. Will try to take a look and possibly use that.